app.py

```
@@ -0,0 +1,26 @@
+from flask import Flask, request, jsonify, render_template, send_from_directory
```

Ruff (0.8.2)

`flask.jsonify` imported but unused

Remove unused import: `flask.jsonify`

(F401)